tests/integration/components/frost-date-picker-test.js

```diff
@@ -99,6 +99,30 @@ describe(test.label, function () {
     })
   })
 
+  describe('when empty string is provided', function () {
```

Comment:
I do not see tests for `undefined` or `null`.

Reply:
Undefined is tested directly above it. I didn't test null because I thought that was overkill. If you would like to see it, I will gladly add it for you.

Reply:
Yes, please add the test for `null`. The test above that is for `undefined` is incorrect. What we would want in that case is to not set the property at all so that we can validate that it is not being set to something else in that case.